### ISSUE TITLE
Re-export models from util-dynamodb in lib-dynamodb

### ISSUE THREAD

**Issue by trivikr** [aws/aws-sdk-js-v3#5048]
### Describe the feature

Re-export models from util-dynamodb in lib-dynamodb

### Use Case

Anyone who wants to use Native AttributeValue types from lib-dynamodb has to add dependency to util-dynamodb.

For example:
```js
import { ScanCommandInput } from "@aws-sdk/lib-dynamodb";
import { NativeAttributeValue } from "@aws-sdk/util-dynamodb";

const TableName: string = "tableName";
const ExclusiveStartKey: Record<string, NativeAttributeValue> = { id: "exclusiveStartKey" };

const scanCommandInput: ScanCommandInput = { TableName, ExclusiveStartKey };
```

### Proposed Solution

Re-export models from util-dynamodb in lib-dynamodb, so the import from util-dynamodb is not required.

Anyone who wants to use Native AttributeValue types from lib-dynamodb has to add dependency to util-dynamodb.

For example:
```js
import { NativeAttributeValue, ScanCommandInput } from "@aws-sdk/lib-dynamodb";

const TableName: string = "tableName";
const ExclusiveStartKey: Record<string, NativeAttributeValue> = { id: "exclusiveStartKey" };

const scanCommandInput: ScanCommandInput = { TableName, ExclusiveStartKey };
```

### Acknowledgements

- [X] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### SDK version used

3.382.0

### Environment details (OS name and version, etc.)

Node

**Comment by Ishmaello** [comment]
Some runs fail response requires migrate code for AWS SDK for JavaScript v3.is this the issue to put

**Comment by trivikr** [comment]
> Some runs fail response requires migrate code for AWS SDK for JavaScript v3

I don't understand your request.

If you need support with respect to codemod, please create/upvote issues on https://github.com/awslabs/aws-sdk-js-codemod/issues
If you have any problems with v2, please create/upvotes issues on https://github.com/aws/aws-sdk-js/issues
If you have problems with v3, please create/upvote issues on https://github.com/aws/aws-sdk-js-v3/issues

**Comment by Ishmaello** [comment]
Run aws-actions/configure-aws-credentials@v2
Error: Input required and not supplied: aws-region
(node:1703) NOTE: We are formalizing our plans to enter AWS SDK for JavaScript (v2) into maintenance mode in 2023.

Please migrate your code to use AWS SDK for JavaScript (v3).
For more information, check the migration guide at https://a.co/[7](https://github.com/Ishmaello/polygon-edge/actions/runs/5779793298/job/15662572723#step:3:8)PzMCcy
(Use `node --trace-warnings ...` to show where the warning was created

**Comment by trivikr** [comment]
```console
Error: Input required and not supplied: aws-region

(node:1703) NOTE: We are formalizing our plans to enter AWS SDK for JavaScript (v2) into maintenance mode in 2023.
```

Looks like this issue is specific to JS SDK v2.
Please report it on https://github.com/aws/aws-sdk-js/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc, if not already reported.

**Comment by Ishmaello** [comment]
Thanks

**Comment by andre-araujo** [comment]
This issue seems to be resolved with https://github.com/aws/aws-sdk-js-v3/pull/5982/commits. Anything left @trivikr?

**Comment by andre-araujo** [comment]
This issue seems to be resolved with https://github.com/aws/aws-sdk-js-v3/pull/5982/commits. Anything left @trivikr?

**Comment by trivikr** [comment]
> This issue seems to be resolved with [#5982 (commits)](https://github.com/aws/aws-sdk-js-v3/pull/5982/commits).

Yes. Thanks of the verification.

> Anything left @trivikr?

No. I verified that `NativeAttributeValue` can now be imported from `@aws-sdk/lib-dynamodb` since https://github.com/aws/aws-sdk-js-v3/releases/tag/v3.552.0.
